### ISSUE TITLE
Super-majority for Appeal

### ISSUE THREAD

**Issue by ChristopherA** [w3c/w3process#131]
I don't know of any other major standards organization where in the process for final appeals, that the final vote is a simple majority. I don't believe true consensus is required, but I do believe that at this level greater then simple majority is required, otherwise more consensus building should be required.

**Comment by dwsinger** [comment]
see also #48 and https://www.w3.org/wiki/Process2018#.2348_The_appeals_process_doesn.27t_actually_describe_how_an_appeal_finishes_2

**Comment by michaelchampion** [comment]
@ChristopherA  what supermajority threshold would you suggest? 2/3 perhaps?

**Comment by ChristopherA** [comment]
I guess I'm seeking that if a topic moves to a the point of a major appeal (either to an AC decision or a Director's decision), that a greater common consensus should be attempted before a change is made to the status quo before the appeal. This can be in the form of seeking greater quorum, or a super-majority (say 67%) without the greater quorum.  The ideal should be that very small dedicated intransigent minority should not get always get their way (say 5-10%?), but that if there is a large minority group of people dissatisfied with a decision (11-33%?) then further discussion and seeking of greater consensus is required.

**Comment by ChristopherA** [comment]
In the case of IETF, the ultimate appeal is not a large group, but the IAB, which according to rfc2026 "review the situation and attempt to resolve it in a manner of its own choosing". However, in practice, like the rest of the IETF, a "rough consensus" is sought by the IAB members.

In Oasis, there are TWO votes if there is an appeal, but then it is the board that decides (and doesn't like IETF say how the board decides).

**Comment by ChristopherA** [comment]
In ANSI, it is also a final appeals board, which does decide based on a majority vote. https://share.ansi.org/Shared%20Documents/Standards%20Activities/American%20National%20Standards/Procedures%2C%20Guides%2C%20and%20Forms/2017-ANSI-Appeals-Board.pdf

**Comment by dwsinger** [comment]
Christopher also commented in AC Review, placed here for the record: "Blockstream does not endorse Section 7.2 regarding appeal of Director's decision. Instead we support that greater consent is required or more concerted efforts (multiple rounds) to address a significant minority's concerns. We do not formally object to adoption of this document."

**Comment by frivoal** [comment]
There is no evidence that appeals are excessively easy to win. Actually, much to the contrary, since no appeal has ever succeeded in the history of W3C. It does not seem that making appeals harder to win is an urgent problem.

Also, last time in happened, a number of the people said (paraphrasing) that they had no strong position on the issue, but did have trust in the director in general, were taking the vote as a vote of (no)confidence, and supported the director. This behavior, which I expect to be typical, already puts a fairly high hurdle on the objectors. Not arguing it is unfair, just that it is. So 49% against the director's decision, 51% for the director's decision, may actually break down in something like 49% against the decision, 25% for the decision, 26% no caring much about the decision but supporting the director in general.

In other words, as I see it, social dynamics mean we already have a super majority of sorts bundled in, and raising the bar would essentially make appeals impossible.

**Comment by ChristopherA** [comment]
As I said in my statement to the AC that either a greater consent (total % of voting, or higher quorum requirement as many members did not vote), or more rounds to reach consent, or all three

**Comment by dwsinger** [comment]
I'm not sure, but I suspect that we may have two questions here:
a) should it need a super-majority to OVERTURN a Director's decision?
b) should it need a super-majority to SUPPORT a Director's decision?

I only asked (a), and @frivoal  points out (correctly, IMHO) that it may be unneeded. @ChristopherA do you mean (a) or (b)?

**Comment by chaals** [comment]
I'm not especially keen on super-majority requirements. If we do so, I would suggest that unless there is a super-majority outcome the issue should be returned as undecided - appeals are pretty rare, and anything that can generate less than a super majority as an appeal outcome may well be something that really does need more work in some area.

**Comment by dwsinger** [comment]
Ah, so three possible uses of super-majority:
1) 'trust the director': it requires a super-majority of the AC to overturn a director's decision
2) 'require consensus': it takes a super-majority of the AC to confirm a director's decision
3) 'require clarity': both 1 & 2, and if neither happen, the question is undecided.

I cannot work out what happens to the decision if the vote is undecided; it's re-asked? If the decision stands or is reversed, then we're in 1 or 2...

**Comment by chaals** [comment]
> I cannot work out what happens to the decision if the vote is undecided; it's re-asked?

The decision presumably is rescinded. If the subject of the decision is a spec it could then be returned to the WG for further work, with the Director clarifying that another request for transition will need to show what changed to be entertained favourably. The Director can equally re-ask the question hoping for a different response which might be more likely to occur after further discussion, or direct W3C to abandon the work, also under the existing process.

**Comment by dwsinger** [comment]
If the decision is rescinded, then we're in case 2 I outlined: it takes a super-majority to confirm a director's decision.

**Comment by chaals** [comment]
I took 3 as a starting point - giving us 3 states: A decision is clearly rejected, or clearly confirmed, or the outcome is unclear. I think there is a case for stating that if a decision is overturned then there should be more clear guidance about what to do - e.g. The Director is obliged to return the spec to the WG or abandon it. In the middle case, I think we should be less prescriptive about how to resolve the question, for example allowing for the possibility of asking the question again, presumably following discussion that suggests the outcome might change.

This is for a pretty unusual situation to begin. 

Yet another alternative is that a repeat of the question requires a higher level of support/rejection, giving the director more latitude to take the range of options - or conversely a lower threshold like simple majority, which merely requires the director to face the opprobrium of e.g. an unhappy 49% for accepting an outcome that clearly had no consensus, relying on social pressure to motivate careful thinking about how to approach resolution.

I'm with @michaelchampion (and I suspect many more) that at this point, voting is a pretty crude mechanism for reaching a decision. Given the Consortium's overwhelming bias for consensus, the "social pressure" to find a reasonable outcome may be sufficient to keep us from having a second round of voting until there is a clear outcome with at least rough consensus...

**Comment by michaelchampion** [comment]
This is quite a bit more gnarly than I thought.  I assumed "supermajority" meant #1; #2 never occurred to me, and I suspect it's unprecedented in any real-world organization whose founder retains a lot of formal authority.  Nevertheless, @ChristopherA has a point that consensus to change the status quo  implies more than a 50% majority.  Given there are sensible arguments each way, and a can of worms in the middle, I'd prefer to close this issue until we have evidence that this is a real problem, and have more than one appeal to try to learn from to solve it.

**Comment by dwsinger** [comment]
so, this seems awfully complex. I think you end up with four states:
* strong confirm: over super-majority confirming
* weak confirm: over 50% but below super-majority confirming
* weak reject: over 50% but below super-majority rejecting
* strong reject: over super-majority rejecting

for something we hope won't happen again and is unlikely to recur at all, I think this is waaaaay too complicated.

**Comment by nrooney** [comment]
Nah the four states seems fine to me. Logic holds.

**Comment by dwsinger** [comment]
I don't see any logic in making it anything other than the final binary decision it is now. Making the 'final appeal' not-final makes life way too complex.

**Comment by dwsinger** [comment]
The process CG feels that this is a tarpit, and since we have had only one appeal in the entire history of the W3C, we risk spending huge amounts of time and effort on something that rarely happens. We'd rather not go here.

**Comment by frivoal** [comment]
@ChristopherA I am preparing a Disposition of Comments for this year's iteration on the Process. It was decided not to address this in Process 2019. Since you raised the initial issue, I would appreciate if you can let us know if this is something you can live with, or if you wish to object to.
